### PR TITLE
staging package build

### PR DESCRIPTION
**Pull request by fivetran-jamie**

### CODE REVIEW

File: integration_tests/dbt_project.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    page_rule_history: "{{ ref('page_rule_history') }}"
+    user: "{{ ref('user') }}"
+    visitor_history: "{{ ref('visitor_history') }}"
+
+    pendo__event_pass_through_columns: []
+    pendo__feature_event_pass_through_columns: []
+    pendo__page_event_pass_through_columns: []
+    pendo__account_history_pass_through_columns: []
+    pendo__visitor_history_pass_through_columns: []
+
```
Comment (fivetran-joemarkiewicz) [inline integration_tests/dbt_project.yml:31]: For spark to dispatch effectively we will want to add the dispatch configuration to our integration tests.
```suggestion
dispatch:
  - macro_namespace: dbt_utils
    search_order: ['spark_utils', 'dbt_utils']
```

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
@fivetran-jamie great job putting this together so quick! I added a few comments and requests for minor formatting changes to address before release. Additionally, I know you had mentioned wanting to update our PR and Issue templates. Was there anything you could think of that we could pilot in this repo?

Once you make the changes it would probably be easiest to just re-request review once you are able to add the README and yml documentation.

File: models/stg_pendo__user.sql [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+final as (
+    
+    select 
+
+        id as user_id,
+        deleted_at,
+        first_name,
+        last_name,
+        role,
```
[the remark this answers is not in the document]
Comment (fivetran-joemarkiewicz) [reply models/stg_pendo__user.sql:30]: This field being `role`. The comment didn't make it super apparent haha

File: macros/get_page_history_columns.sql [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{% macro get_page_history_columns() %}
```
[the remark this answers is not in the document]
Comment (fivetran-jamie) [reply macros/get_page_history_columns.sql:1]: will ask drew...

File: models/tmp/stg_pendo__account_history_tmp.sql [hunk 4]
```diff
@@ -0,0 +1 @@
+select * from {{ var('account_history') }}
```
[the remark this answers is not in the document]
Comment (fivetran-jamie) [reply models/tmp/stg_pendo__account_history_tmp.sql:1]: ah yeah we should adjust the bash script to automatically align with this

**Pull request merged**